modules/ROOT/pages/cbl-whatsnew.adoc

```diff
@@ -17,6 +17,33 @@ NOTE: Couchbase Lite 3.0 introduces some breaking changes. +
 If you are upgrading from 2.x, please refer to the appropriate upgrade page -- see: <<lbl-upgrade>>
 Users should be able to upgrade to 3.1.x from 3.0.x without manual intervention.
 
+== New JavaScript Plugins for Couchbase Lite
```

Comment:
@ElliotFrancisHunter  Please remove entire  Whats New Section related to plugins. We do not include release notes for couchbase community projects in our official docs and the plugin releases are not directly  linked to CBL releases. Any release notes will be part of that plugin documentation. 
Moot point but the the content is also wrong - The plugins are not enterprise

Reply:
@rajagp Done, I have some confusion here though. The plugins themselves require CBL with an enterprise license according to the doc so what differentiates requiring an enterprise version of CBL with being enterprise only?